packages/@o3r/telemetry/src/schematics/index.ts

```diff
@@ -2,6 +2,7 @@ import { callRule, Rule } from '@angular-devkit/schematics';
 import { performance } from 'node:perf_hooks';
 import { lastValueFrom } from 'rxjs';
 import { getEnvironmentInfo } from '../environment/index';
+import { SchematicMetricData, sendData } from '../sender';
```

Comment:
would be graat to have an option parameter to `createSchematicWithMetrics` to provide a custom `sendData` function to target different log server if needed